### ISSUE TITLE
Keyboard focus is not visible at Hamburger menu present at the UI.

### ISSUE THREAD

**Issue by msftedad** [jhildenbiddle/docsify-themeable#86]
**Test Environment:**

**Steps to reproduce**

1. Open the PWA Builder URL-https://docs.pwabuilder.com/#/ in Edge browser.
2. PWA Builder home page will appear.
3. Navigate on the Hamburger menu with keyboard.
4. Observe the Keyboard focus.

**Current behaviour**
Keyboard focus is not visible at Hamburger menu present at the UI.

**Expected behaviour**
Proper keyboard focus must be visible for the Hamburger menu present at the UI.

**Attachments:**

https://github.com/jhildenbiddle/docsify-themeable/assets/93735775/f4610715-8733-466a-bd71-492520b7395c

![286163286-07b6d273-c840-4a7a-933e-8deb338cfd99](https://github.com/jhildenbiddle/docsify-themeable/assets/93735775/a11188f7-f60b-4baf-a538-3d20e0f2549b)
